package.json

```diff
@@ -20,7 +20,7 @@
     "start": "node --experimental-modules -r dotenv/config ./modules/node_modules/@frogpond/ccc-server/index.mjs",
     "stolaf": "env INSTITUTION=stolaf-college npm start",
     "carleton": "env INSTITUTION=carleton-college npm start",
-    "test": "echo \"Error: no test specified\""
+    "test": "./scripts/smoke-test.sh"
```

Comment:
I prefer the dirname `script`.

Reply:
I would rather keep it as `scripts`…

Reply:
And I would rather it be named `script`.

Reply:
But it's a folder of "scripts", not a folder of "script".  It should be pluralized.

Reply:
@hawkrives @rye can we settle on a name?

Reply:
`script` is still my favorite

Reply:
We're going on 22 days strong over a singular versus a plural. Please, one of you... just... please. Pretty please.

Reply:
How about… I move the file to the "correct" location as I see it… and then we take it from there. Otherwise @hawkrives can just merge with one disapproving review.